javascript/javascript2/week1/movieApp/features.js

```diff
@@ -154,10 +184,43 @@ function displayMovies(movies) {
     heroSection.appendChild(noMoviesFoundPara);
   }
 }
+function moviesByYear() {
+  const sortedMoviesbyYear = movieList.sort(
+    (itemA, itemB) => itemB.movie_year - itemA.movie_year
+  );
+  return sortedMoviesbyYear;
+}
+function moviesbyHighRating() {
+  const sortedMoviesbyRating = movieList.sort(
+    (itemA, itemB) => itemB.rating - itemA.rating
+  );
+  return sortedMoviesbyRating;
+}
+function sciFiMovie() {
```

Comment:
Naming your functions or variables with a plural name helps to understand that it returns an array of values.

```suggestion
function sciFiMovies() {
```